src/bin/elfshaker/status.rs

```diff
@@ -102,28 +172,37 @@ fn probe_snapshot_files(
                     path.display()
                 );
                 true
+            } else if workspace_is_symlink {
```

Comment:
This "workspace_is_" naming scheme that was already present if weird. I would at least say : "file_in_workspace_is_"

Reply:
done